src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -379,7 +379,7 @@ A working alternative would be using an icon together with an `aria-label`:
 			ref="sidebar"
 			class="app-sidebar"
 			:aria-labelledby="`app-sidebar-vue-${uid}__header`"
-			@keydown.esc.stop="isMobile && closeSidebar()">
+			@keydown.esc="isMobile && ($event.stopPropagation() | closeSidebar())">
```

Comment:
Is it intended to use "bitwise Or" here?

If it is to call to independent actions, let's move an entire handler to a method